packages/kernel-5.10/kernel-5.10.spec

```
@@ -83,6 +86,12 @@ Summary: Modules for the Linux kernel
 %description modules
 %{summary}.
 
+%package mkfs-confs
```

Suggestions to improve the dependencies:
1. Require the parent package so that the installed configs match the installed kernel.
2. Provide a generic capability (using parens, by convention) that other packages can key off of
3. Conflict with any other package providing the same capability

Goal is to make it so that only one `mkfs-confs` package can be installed.

```suggestion

%package mkfs-confs
Requires: %{name}
Provides: %{_cross_os}kernel(mkfs-confs)
Conflicts: %{_cross_os}kernel(mkfs-confs)
```

However, that said - I would not make this a separate subpackage, but instead add that `Provides` and `Conflicts` to the parent kernel package.